features/user/managing_customers/seeing_customer_details.feature

```diff
@@ -0,0 +1,24 @@
+@managing_customers
+Feature: Seeing customer's details
+    In order to see customer's details in the store
+    As an Administrator
+    I want to be able to show specific customer's page
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has customer "f.baggins@shire.me" with name "Frodo Baggins" since "2011-01-10 21:00"
+        And he has shipping address as "Hobbiton", "Bag End", "1", "New Zealand" for "Frodo Baggins"
+        And he has billing address as "Rivendell", "The Last Homely House", "7", "New Zealand" for "Bilbo Baggins"
```

Comment:
I would move the default billing and shipping address setup to `Given` for the second scenario only (`Seeing customer's addresses`). This will also ensure that the first scenario (customer without addresses) will display correctly and do not 💥 .